### DISCUSSION TITLE
OKD 4.7 - no route to host

### DISCUSSION THREAD

**Discussion by indacofilippo** [okd-project/okd#1099] [Q&A]
When installing an okd 4.7 lab, following the guide https://itnext.io/guide-installing-an-okd-4-5-cluster-508a2631cbee by @cragr  (Craig Robinson), the start phase of the workers and masters stops with error "_Get error: Get "https: //api-int.lab.domain: 22623 / config / master" dial tcp 192.168.1.210:22623: connect: no route to host_"
I have done several checks, I am attaching screenshots, but I cannot understand where the problem is.
Do you have any suggestions? Has something similar happened to you?
Thanks in advance for your help.

![okd-error-install](https://user-images.githubusercontent.com/7949159/152716897-29b78b52-22fa-4e62-887c-67f95ae2d36c.PNG)

![okd-noroute-01](https://user-images.githubusercontent.com/7949159/152716664-32892219-3354-4c07-a7e7-95fb3795a8df.PNG)

![okd-noroute-worker](https://user-images.githubusercontent.com/7949159/152716675-77b2d7ee-81eb-430a-baa5-9e132d8b7b08.PNG)

![okd-ping-hosts](https://user-images.githubusercontent.com/7949159/152716665-5842b252-961d-4ca5-9309-d799dd5c7c18.PNG)

![okd-dhcp-leases](https://user-images.githubusercontent.com/7949159/152716666-f5acc9de-3931-4e9c-9005-faedee9d2f81.PNG)

![okd-netstat](https://user-images.githubusercontent.com/7949159/152716669-0c738b20-bbcf-4110-b8cf-6a322807e2be.PNG)

![okd-ping](https://user-images.githubusercontent.com/7949159/152716670-eee19b9d-9013-4380-ab1b-5c3da35d8068.PNG)

![okd-timecurl-01](https://user-images.githubusercontent.com/7949159/152716673-27fa5b58-8084-4f17-9439-3d660a84c687.PNG)
![okd-timecurl-02](https://user-images.githubusercontent.com/7949159/152716671-d61a0c6f-b855-4b4d-a1d0-bac23e1ce859.PNG)

![okd-telnet-22623](https://user-images.githubusercontent.com/7949159/152716674-b2c3d973-436e-4c7c-a7ac-0d77f853e096.PNG)

**Comment by indacofilippo** [comment]
I notice that the haproxy GUI shows me down both the masters and the workers: is this normal?

![okd-stats-haproxy](https://user-images.githubusercontent.com/7949159/152718760-88702409-a293-4f95-9f53-aa009c4ecedd.PNG)

**Comment by cniackz** [comment]
I have the exact same problem @indacofilippo 

```
[root@3c-ec-ef-1d-75-fc core]# curl -k https://65.49.37.0:22623/config/master
curl: (7) Failed to connect to 65.49.37.0 port 22623: No route to host
```

I have been reading at https://access.redhat.com/solutions/4926401 to understand.
And it seems this server runs per master node.
In my case, installation was done in one master but not others.
So I want to know why isn't running now? and how to run this process manually to complete the installation?

Anyway, if you happen to know the solution by now, please let me know!!!

416 830 7619
celis.hernandez.cesar@gmail.com

Thank you 👍
